utility_scripts/compare_models.py

```diff
@@ -305,59 +313,58 @@ def prepare_plots(self, model_1_score_path, model_1_label_path, model_2_score_pa
+
+        self._prepare_plots(self.fig_auc, 'Area Under Receiver Operator Curve', figure_supertitle, figsize)
+
+        self._prepare_plots(self.fig_roc, 'Receiver Operator Curves', figure_supertitle, (10, 15))
+
+        self._prepare_plots(self.fig_afb, 'Allele Frequency bins performance comparison', figure_supertitle, (11, 11))
+
+        self._prepare_plots(
+            self.fig_score_dist_box, 'raw CAPICE score distributions', f'{figure_supertitle}{post_supertitle}', figsize
         )
-        self.fig_roc.set_constrained_layout(constrained_layout)
-        self.fig_afb = plt.figure(figsize=(11, 11))
-        self.fig_afb.suptitle(
-            f'Model 1 vs Model 2 Allele Frequency bins performance comparison\n'
-            f'Model 1 scores: {model_1_score_path}\n'
-            f'Model 1 labels: {model_1_label_path}\n'
-            f'Model 2 scores: {model_2_score_path}\n'
-            f'Model 2 labels: {model_2_label_path}\n'
+
+        self._prepare_plots(
+            self.fig_score_dist_vio, 'raw CAPICE score distributions', figure_supertitle, figsize
         )
-        self.fig_afb.set_constrained_layout(constrained_layout)
-        self.fig_score_dist = plt.figure(figsize=figsize)
-        self.fig_score_dist.suptitle(
-            f'Model 1 vs Model 2 raw CAPICE score distributions\n'
-            f'Model 1 scores: {model_1_score_path}\n'
-            f'Model 1 labels: {model_1_label_path}\n'
-            f'Model 2 scores: {model_2_score_path}\n'
-            f'Model 2 labels: {model_2_label_path}\n'
-            f'(M1B = Model 1 Benign, M1P = Model 1 Pathogenic, M2B = Model 2 Benign, M2P = Model 2 '
-            f'Pathogenic)\n'
+
+        self._prepare_plots(
+            self.fig_score_diff_box,
+            'absolute score differences to the true label',
+            f'{figure_supertitle}{post_supertitle}',
+            figsize
         )
-        self.fig_score_dist.set_constrained_layout(constrained_layout)
-        self.fig_score_diff = plt.figure(figsize=figsize)
-        self.fig_score_diff.suptitle(
-            f'Model 1 vs Model 2 absolute score difference to the true label\n'
-            f'Model 1 scores: {model_1_score_path}\n'
-            f'Model 1 labels: {model_1_label_path}\n'
-            f'Model 2 scores: {model_2_score_path}\n'
-            f'Model 2 labels: {model_2_label_path}\n'
-            f'(M1B = Model 1 Benign, M1P = Model 1 Pathogenic, M2B = Model 2 Benign, M2P = Model 2 '
-            f'Pathogenic)\n'
+
+        self._prepare_plots(
+            self.fig_score_diff_vio,
+            'absolute score differences to the true label',
+            figure_supertitle,
+            figsize
         )
-        self.fig_score_diff.set_constrained_layout(constrained_layout)
+
         print('Plot figures prepared.\n')
 
+    @staticmethod
+    def _prepare_plots(
+            figure: plt.Figure,
+            plot_type: str,
+            figure_supertitle: str,
```

Comment:
Is there a specific benefit on accepting this as separate input instead of joining them when calling this method (except maybe a single `\n` that doesn't need to be duplicated)?

For example: With the boxplot, the second parameter ends up being more complex anyway (`'raw CAPICE score distributions', f'{figure_supertitle}{post_supertitle}'`), so might as well simply require one parameter (`f'raw CAPICE score distributions\n{figure_supertitle}{post_supertitle}'`)